frontend/src/components/historyTable/Table.styles.js

```diff
@@ -16,17 +16,20 @@ export const TableStyled = styled.table`
     }
 
     thead th {
-        padding: 10px;
+        padding: 8px;
         text-align: left;
-        vertical-align: middle;
     }
 
+    /* thead th:hover {
```

Comment:
Is this still supposed to be a comment?

Reply:
yes, but i will remove this ugly comment from the code